packages/common/src/api/tan-query/utils/primeCollectionData.ts

```diff
@@ -17,14 +17,20 @@ export const primeCollectionData = ({
   collections,
   queryClient,
   dispatch,
-  forceReplace = false
+  forceReplace = false,
+  skipQueryData = false
```

Comment:
feels like this util is now doing two jobs with somewhat different intents, but that's ok for now until we figure out the cacheing plan